mix.exs

```diff
@@ -58,9 +65,15 @@ defmodule Cassette.Mixfile do
       {:ex_doc, "~> 0.11", only: :dev},
       {:earmark, "~> 1.0", only: :dev},
       {:bypass, "~> 1.0", only: [:dev, :test]},
-      {:credo, "~> 1.0", only: [:dev, :test], runtime: false},
       {:fake_cas, "~> 1.1", only: [:dev, :test]},
       {:excoveralls, "~> 0.7", only: :test}
     ]
+    |> (fn deps ->
+          if Version.compare(@elixir_version, @min_version_for_credo) in [:gt, :eq] do
```

Comment:
WDYT adding to do telling when it can be removed?

Reply:
Do you think it is clear now?